proposals/lip-0044.md

```diff
@@ -516,59 +598,22 @@ getGeneratorsBetweenTimestamps(startTimestamp, endTimestamp):
 
 #### setGeneratorList
```

Comment:
I think it would be nicer to have code here to make also the exception explicit as it has implications for the state transitions.

Reply:
done